spec/pio/open_flow13/match_spec.rb

```diff
@@ -52,6 +52,51 @@
       And { match.match_fields[0].oxm_length == 6 }
     end
 
+    context "with metadata: 1" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.